tests/test_code_gen.py

```diff
@@ -611,6 +611,24 @@ def test_annassign(self):
         """
         self.assertAstRoundtripsGtVer(source, (3, 6))
 
+    @unittest.skipUnless(sys.version_info >= (3, 6),
+                         "typing and ann assignment was introduced in "
```

Comment:
Is "ann" short for "annotation" here? If so, it would be nice to use the longer version.

Reply:
I guess it is named after the `AnnAssign` node.

Reply:
"ann" was short for "annotated".  Fixed in rvprasad/astor@78e7c60a0.